base_example/tower_configs/tower_settings_individuale.yml

```diff
@@ -0,0 +1,46 @@
+---
+tower_settings_individuale:
```

Comment:
Is there a reason this reads `individuale` rather than `individual`?

Reply:
As well as the filename, and the usage in the playbook

Reply:
I think we arent using this, and its this way in our normal tower_configuration, I think autocorrect decided to have a field day.

Reply:
Removed